.github/workflows/dockerfile-health.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: "Dockerfile Health"
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - .devcontainer/Dockerfile
+      - .github/workflows/dockerfile-health.yml
+      - lte/gateway/docker/mme/Dockerfile.ubuntu20.04
+
+jobs:
+  build_ubuntu20_04_dockerfile:
+    env:
+      DOCKERFILE: lte/gateway/docker/mme/Dockerfile.ubuntu20.04
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Check Out Repo
+        uses: actions/checkout@v2
+      -
+        name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ hashFiles( '**/Dockerfile*' ) }}
+      -
+        name: Docker Build
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ${{ env.DOCKERFILE }}
+          push: false
+          tags: magma/mme_builder:latest
```

Comment:
I know it's only a docker build test, but as a good practise let's double tag with latest and version (git-hash0:8)

Reply:
I'll explore function support - I did a brief google-fu earlier and didn't find but it must support better re-use. I'll look harder this time =D

Reply:
Huh. Not great. Appears my early google-fu was correct and GH Actions don't yet support any form of re-use?
https://github.community/t/reusing-sharing-inheriting-steps-between-jobs-declarations/16851

Reply:
Should we start bundling them in our own Github Action marketplace "orbs"?